packages/terra-button-group/tests/wdio/button-group-spec.js

```diff
@@ -1,109 +1,153 @@
 /* global before, browser, Terra */
 
 describe('Button Group', () => {
-  describe('Tiny', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/tiny-button-group'));
+  describe('Text Button', () => {
+    beforeEach(() => browser.url('/#/tests/button-group-tests/text-button-group'));
     Terra.should.beAccessible();
     Terra.should.matchScreenshot();
   });
 
-  describe('Small', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/small-button-group'));
+  describe('Icon Button', () => {
+    beforeEach(() => browser.url('/#/tests/button-group-tests/icon-button-group'));
     Terra.should.beAccessible();
     Terra.should.matchScreenshot();
   });
 
-  describe('Medium', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/medium-button-group'));
+  describe('Not Selectable', () => {
+    beforeEach(() => browser.url('/#/tests/button-group-tests/not-selectable-button-group'));
     Terra.should.beAccessible();
     Terra.should.matchScreenshot();
   });
 
-  describe('Large', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/large-button-group'));
+  describe('Disabled Buttons', () => {
+    beforeEach(() => browser.url('/#/tests/button-group-tests/not-selectable-button-group'));
     Terra.should.beAccessible();
     Terra.should.matchScreenshot();
   });
 
-  describe('Huge', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/huge-button-group'));
-    Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
-  });
+  describe('onChange', () => {
+    before(() => browser.url('/#/tests/button-group-tests/on-change-button-group'));
+    it('should select the first button', () => {
+      browser.keys('Tab');
+      browser.keys('Space');
+      expect(browser.getText('#selected-key')).to.equal('1');
+    });
+
+    Terra.should.themeEachCustomProperty({
+      '--terra-button-group-active-background-color': 'white',
+      '--terra-button-group-active-and-hover-background-color': 'white',
+      '--terra-button-group-active-and-focus-background-color': 'white',
+    });
 
-  describe('Compact', () => {
-    beforeEach(() => browser.url('/#/tests/button-group-tests/compact-button-group'));
     Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
-  });
+    Terra.should.matchScreenshot('0');
```

Comment:
I noticed that a few of the wdio tests for button-group pass a number to `Terra.should.matchScreenshot`, what is it used for?

Reply:
Each one of these wdio tests involves a series of clicks and take screenshots along the way. The parameter passed to the `Terra.should.matchScreenshot` helper is to give it a custom name for the screenshots. See this [comment](https://github.com/cerner/terra-toolkit/pull/37#issuecomment-355068062). I have changed the parameter to give it a more descriptive name: https://github.com/cerner/terra-core/pull/1155/commits/9d6dd46edcf8e4be2a28ff8717995749d8cc8dde